services/frontend/src/components/block-producer-card.js

```diff
@@ -40,18 +42,20 @@ const styles = theme => ({
     justifyContent: 'space-between'
   },
   radar: {
-    background: theme.palette.primary.dark,
+    background: theme.palette.surface.light,
     paddingTop: theme.spacing(2),
     paddingBottom: theme.spacing(2)
   },
   avatar: {
-    backgroundColor: theme.palette.primary.main
+    backgroundColor: theme.palette.primary.light
```

Comment:
The background color for the avatar should be replaced to match the same background color as the containing element. 

The dark  grey contrast of `#787291` does not look well with the Block producer logos

![image](https://user-images.githubusercontent.com/5632966/76045646-5dc66a80-5f23-11ea-9725-0a371c6956ae.png)